src/components/card/Card.tsx

```diff
@@ -0,0 +1,36 @@
+import Image from "next/image";
+import { useState } from "react";
+import Button from "./Button";
+
+type ICard = {
+    title: string;
+    description?: string | JSX.Element;
+    cover?: string;
+    waterAmount?: number;
+}
+
+const Card = (props: ICard) => {
```

Comment:
Another small UX suggestion -- you could open the "details" hover when hovering over the whole component, not just the details part.

Reply:
Do you have a suggestion how to do it with Tailwind?

Reply:
Think I formulated the comment wrong, sorry!

Do you think it'd be useful to have this state as default? Without the hover state. Maybe add text truncation if the text overflows too much. 
<img width="294" alt="image" src="https://user-images.githubusercontent.com/681522/207286407-03420a70-4214-4d0c-bf46-5dc43f873f08.png">